tests/contrib/gevent/test_tracer.py

```diff
@@ -144,8 +144,8 @@ def green_2():
         worker_2 = spans[2]
         # check sampling priority
         assert parent_span.get_metric(SAMPLING_PRIORITY_KEY) == USER_KEEP
-        assert worker_1.get_metric(SAMPLING_PRIORITY_KEY) == USER_KEEP
```

Comment:
This needs to be fixed still.

Reply:
I don't think there's a good existing way to solve this - it gets super messy to have to copy the tags from span to span when partial flushing, forking, etc.

Reply:
Solution: store trace-level tags on `Context`.

Also looking at this test case again the change is warranted. Before, the context needed to be cloned and passed the worker which would add the sampling priority tags to the worker span even though they weren't necessary. 
Now, there is no need to attach these tags because the trace will be combined.